content/Developer_Program/WebAuthn_Starter_Kit/Identifier_First_Flow.adoc

```diff
@@ -11,7 +11,7 @@ In the identifier first flow, a user is only prompted to enter their username, a
 
 One of the potential avenues of attack against WebAuthn is the theft or misuse of a hardware authenticator. To secure against these physical attacks, WebAuthn has the option to require *User Verification (UV)*, namely having the user demonstrate that they are the authorized user for the WebAuthn Authenticator via an authenticator PIN or biometric identification like a fingerprint. Various WebAuthn authenticators can support different levels of UV, depending on the web browser and host Operating System (OS). The documentation will refer to the case where UV is supported on the authenticator (UV=1) as *User Verified* authentication or registration.
 
-In order to support all authenticators without compromising security, the environments and authenticators where UV actions are not supported must be taken into account. These cases are referred to as *Server Verified (SV)* authentication and registration. In an SV interaction, the user must still prove that they are the authorized user of the authentication device by supplying a *Server Verified PIN (SV-PIN)*. Unlike a password, the SV-PIN is not directly used to authenticate the user but instead is used only to verify that a user is a legitimate user of the authenticator. This creates an identical user experience to a WebAuthn authenticator secured with an onboard authenticator PIN, and prevents theft of a hardware authenticator from directly compromising an account.
+In order to support all authenticators without compromising security, the environments and authenticators where UV actions are not supported must be taken into account. These cases are referred to as *Server Verified (SV)* authentication and registration. In an SV interaction, the user must still prove that they are the authorized user of the authentication device by supplying a *U2F Password*. Unlike a password, the U2F Password is not directly used to authenticate the user but instead is used only to verify that a user is a legitimate user of the authenticator. This creates an identical user experience to a WebAuthn authenticator secured with an onboard authenticator PIN, and prevents theft of a hardware authenticator from directly compromising an account.
```

Comment:
The *Server Verified (SV)* term here should be reworded to U2F. Likewise, SV could be replaced with U2F

Reply:
Question on this page - Are we still looking to include this page on Identifier First Flow, or are we going to retire it as we're switching to AMFA?

Reply:
Lets retire it in favor of AMFA

Reply:
Good deal - I set the page as "hidden" in the conf.json settings in the folder